tests/snapd

```diff
@@ -87,55 +79,22 @@ lxc profile device add default root disk pool=default path=/
 lxc network create lxdbr0
 lxc profile device add default eth0 nic nictype=bridged parent=lxdbr0 name=eth0
 
-snap set lxd ceph.builtin=true
-
 systemctl restart snap.lxd.daemon.service
 
-for _ in $(seq 60); do
-    lxd waitready --timeout=10 >/dev/null 2>&1 && break
-done
+lxd waitready --timeout=300
 
 # Check that user access works
-if [ "${ID}" != "ubuntu-core" ]; then
-    (
-        set -e
-        usermod -G lxd -a jenkins || true
-        LXC=$(which lxc)
-        su jenkins -c "${LXC} info"
-    )
-fi
+set -e
+test_user="testuser"
+useradd -m ${test_user} -G lxd
+LXC=$(command -v lxc)
+su ${test_user} -c "${LXC} info"
+userdel -r ${test_user} 2>/dev/null || true
 
 # Container creation test
-lxc launch images:alpine/edge unprivileged
-lxc launch images:alpine/edge privileged -c security.privileged=true
-lxc launch images:alpine/edge isolated -c security.idmap.isolated=true
-
-# Storage backends test
-if modprobe -q -n btrfs; then
-    lxc storage create test-btrfs btrfs
-    lxc launch images:alpine/edge -s test-btrfs btrfs
-fi
-
-if modprobe -q -n zfs && ! grep -q ^0.7 < /sys/module/zfs/version; then
-    lxc storage create test-zfs zfs
-    lxc launch images:alpine/edge -s test-zfs zfs
-fi
-
-if [ "${ID}" != "ubuntu-core" ] && [ "${ID}" != "elementary" ] && modprobe -q -n dm_mod; then
-    lxc storage create test-lvm lvm volume.size=25MB
-    lxc launch images:alpine/edge -s test-lvm lvm
-fi
-
-if [ "${ID}" != "ubuntu-core" ] && [ "${VERSION_CODENAME:-""}" != "xenial" ] && [ "${ID}" != "centos" ] && modprobe -q -n rbd; then
-    curl http://canonical-lxd.stgraber.org/config/ceph-snap.sh | sh -eu
-
-    lxc storage create test-ceph ceph "ceph.osd.pool_name=$(mktemp -u snaptest-XXX)" volume.size=25MB ceph.osd.pg_num=1
-    lxc launch images:alpine/edge -s test-ceph ceph
-fi
-
-if [ "${track}" != "3.0" ]; then
-    lxc query /1.0/resources
-fi
+lxc launch "${IMAGE}" unprivileged
+lxc launch "${IMAGE}" privileged -c security.privileged=true
+lxc launch "${IMAGE}" isolated -c security.idmap.isolated=true
```

Comment:
I think those are potentially unneeded now that we have [`tests/container`](https://github.com/canonical/lxd-ci/blob/main/tests/container)

Reply:
Removed container-related checks